src/components/DayPickerRangeController.jsx

```diff
@@ -483,6 +483,10 @@ export default class DayPickerRangeController extends BaseClass {
       startDate = getSelectedDateOffset(startDateOffset, day);
       endDate = getSelectedDateOffset(endDateOffset, day);
 
+      if (this.isBlocked(startDate) || this.isBlocked(endDate)) {
```

Comment:
Was this behavior broken before?

Reply:
It was, I noticed it in making these updates.